pymeasure/instruments/lecroy/lecroyT3DSO1204.py

```diff
@@ -464,6 +465,19 @@ def autoscale(self):
         """ Autoscale displayed channels. """
         self.write("ASET")
 
+    def write(self, command):
+        """ Writes the command to the instrument through the adapter.
+        Note.
+        If the last command was received less than SLEEP_SECONDS before, this method blocks for
+        the remaining time so that commands are never sent with rate more than 1/SLEEP_SECONDS Hz.
+
+        :param command: command string to be sent to the instrument
+        """
+        seconds_since_last_command = time.monotonic() - self._last_command_timestamp
+        if seconds_since_last_command < self.SLEEP_SECONDS:
+            time.sleep(self.SLEEP_SECONDS - seconds_since_last_command)
+        super(LeCroyT3DSO1204, self).write(command)
```

Comment:
you can just say `super().write(command)` afaik.

Reply:
Yes it is true. Applied in commit 2b92a7c1795e42dd8777551329450a2ff8ac6c08